editions/tw5.com/tiddlers/$__StoryList.tid

```diff
@@ -1,5 +0,0 @@
-created: 20210322152203906
-list: [[Documentation Macros]] HelloThere GettingStarted Community
-modified: 20210322152237613
-title: $:/StoryList
-type: text/vnd.tiddlywiki
```

Comment:
@Jermolene @Is this supposed to be deleted? Am I missing something here?

Reply:
I start lots of experimental servers, so sometimes the core editions get Storylists generated when those folders should not have that tiddler... but 8 may have deleted one in error here...